apps/frontend-control/.env.test

```
@@ -13,6 +13,7 @@ APP_ORIGIN_MANAGE=http://127.0.0.1:3002
 APP_ORIGIN_CONTROL=http://127.0.0.1:3003
 APP_ORIGIN_LTI=http://127.0.0.1:4000
 APP_ORIGIN_AUTH=http://127.0.0.1:3010
+APP_ORIGIN_CHAT=http://127.0.0.1:3004
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The APP_ORIGIN_CHAT key should go before the APP_ORIGIN_CONTROL key

(UnorderedKey)